### PR TITLE
Document additional Sentry options in .NET SDK

### PR DESCRIPTION
**Pull request by Tyrrrz**
I'm horrible when it comes to expressing my thoughts and I'm also bad in English, so please help me if there is a better way to formulate this.

### CODE REVIEW

File: src/platforms/common/configuration/options.mdx [hunk 1]
```diff
@@ -258,6 +258,22 @@ Configures a separate proxy for outgoing HTTPS requests. This value might not be
 
 </ConfigKey>
 
+<ConfigKey name="cache-directory-path" supported={["dotnet"]}>
+
+Specifies a local directory used for caching payloads. When this option is enabled (i.e. when the directory is set), Sentry SDK will persist envelopes locally before sending them over to Sentry. This is useful if you expect your application to run in environments where internet connectivity is limited.
```
Comment (PeloWriter) [inline src/platforms/common/configuration/options.mdx:263]:
```suggestion
Specifies a local directory used for cacheing payloads. When this option is enabled (that is, when the directory is set), the Sentry SDK will persist envelopes locally before sending to Sentry. This configuration option is particularly useful if you expect your application to run in environments where internet connectivity is limited.
```

File: src/platforms/common/configuration/options.mdx [hunk 2]
```diff
@@ -258,6 +258,22 @@ Configures a separate proxy for outgoing HTTPS requests. This value might not be
...
+
+Default: not set (caching is disabled).
+
+</ConfigKey>
+
+<ConfigKey name="init-cache-flush-timeout" supported={["dotnet"]}>
+
+When caching is enabled (i.e. <PlatformIdentifier name="cache-directory-path" /> is set), this option controls the timeout which limits how long the Sentry SDK will attempt to flush existing cache during initialization. Note that flushing cache involves sending the payload to Sentry in a blocking operation. Setting this option to zero will mean that Sentry will not attempt to flush cache during initialization, but will do so on the next queued up payload instead.
```
Comment (PeloWriter) [inline src/platforms/common/configuration/options.mdx:271]:
```suggestion
When cacheing is enabled (that is, <PlatformIdentifier name="cache-directory-path" /> is set), this option controls the timeout that limits how long the SDK will attempt to flush existing cache during initialization. Note that flushing the cache involves sending the payload to Sentry in a blocking operation. Setting this option to zero means that Sentry will **not** attempt to flush the cache during initialization, but instead will do so when the next payload is queued up.
```

File: src/platforms/common/configuration/options.mdx [hunk 3]
```diff
@@ -258,6 +258,22 @@ Configures a separate proxy for outgoing HTTPS requests. This value might not be
...
+
+Default: 1 second.
```
Comment (PeloWriter) [inline src/platforms/common/configuration/options.mdx:273]:
```suggestion
The default is `1` (one) second.
```

**Review by PeloWriter — APPROVED** [review]
Minor smoothing.

File: src/platforms/common/configuration/options.mdx [hunk 4]
```diff
@@ -258,6 +258,22 @@ Configures a separate proxy for outgoing HTTPS requests. This value might not be
...
+
+Specifies a local directory used for cacheing payloads. When this option is enabled (that is, when the directory is set), the Sentry SDK will persist envelopes locally before sending to Sentry. This configuration option is particularly useful if you expect your application to run in environments where internet connectivity is limited.
+
+Default: not set (caching is disabled).
+
+</ConfigKey>
+
+<ConfigKey name="init-cache-flush-timeout" supported={["dotnet"]}>
+
+When cacheing is enabled (that is, <PlatformIdentifier name="cache-directory-path" /> is set), this option controls the timeout that limits how long the SDK will attempt to flush existing cache during initialization. Note that flushing the cache involves sending the payload to Sentry in a blocking operation. Setting this option to zero means that Sentry will **not** attempt to flush the cache during initialization, but instead will do so when the next payload is queued up.
```
Comment (bruno-garcia) [inline src/platforms/common/configuration/options.mdx:271]:
```suggestion
When cacheing is enabled (that is, <PlatformIdentifier name="cache-directory-path" /> is set), this option controls the timeout that limits how long the SDK will attempt to flush existing cache during initialization. Note that flushing the cache involves sending the payload to Sentry in a blocking operation. Setting this option to zero means that Sentry will **not** wait for events to be flushed before returning from `Init`, and if the app crashes again before the event is submitted on the background, it's possible events never make it into Sentry for an app that always crashes quickly after startup.
```

**Review by bruno-garcia — APPROVED** [review]

**Pull request merged**